src/full/cache.clj

```diff
@@ -94,6 +94,20 @@
           c))
       (log/warn "Caching is disabled (:memcache-address not configured)"))))
 
+(defn- thaw-val
+  "Since upgrading nippy, some cached values were throwing errors during deserialization
+  To fix that and keep moving with the later version of nippy (and more importantly encore)
+  catch deserialization issues and log them, assuming the caller will simply reload the cache
+  with a freshly reserialized value that should successfully deserialize next time."
+  [k val]
+  (try
+    (nippy/thaw val)
+    (catch Throwable e
+      (do
+        ;; Specifically do not log stack trace since that is slow and noisy and we know it was likely bad bytes
+        (log/debug "Failed to thaw value in cache for " k ", should be refreshed. Error: " (.getMessage e))
```

Comment:
`full.core.log/debug` returns nil, so `do` block is redundant here.

Reply:
Oh yeah duh.